### ISSUE TITLE
Uncaught ReferenceError: module is not defined

### ISSUE THREAD

**Issue by DanielSokil** [radi-js/radi#29]
When loading Radi from CDN, The following error occurs:
```javascript
radi.min.js:58 Uncaught ReferenceError: module is not defined
    at radi.min.js:58
    at radi.min.js:16
    at radi.min.js:16
```

**Comment by Marcisbee** [comment]
Hmm, will check it out right now.

**Comment by Marcisbee** [comment]
Fixed with latest v0.3.29 release.
